pages/electronics-library-dev.tsx

```diff
@@ -18,7 +21,23 @@ interface IElectronicsComponent {
   dataSheet?: string;
   tutorialLinks?: string[];
   additionalLinks?: string[];
-}
+};
+
+enum CategoryTags {
+  ShowAll = "ShowAll",
```

Comment:
Add space between Show and All

Reply:
enum didn't like when I added spaces. Here's the fix I put in. The Tag should have a space now. Good catch!
![image](https://user-images.githubusercontent.com/129784422/231242337-3c3dd1f3-ed33-480d-ab24-58e37baf6d61.png)